README.md

```diff
@@ -8,7 +8,7 @@ If you just want to contribute content, read the [CONTRIBUTING](CONTRIBUTING.md)
 
 ## Cloning
 
-The project installs the [Hugo Relearn Theme](https://mcshelby.github.io/hugo-theme-relearn/) as a git submodule. Thus when cloning
+The project installs the [Hugo Relearn Theme](https://mcshelby.github.io/hugo-theme-relearn/) as a git submodule. Thus, when cloning
```

Comment:
Personally I would not put a comma there, as there is already a comma after _repository_. See also https://english.stackexchange.com/questions/58939/should-so-therefore-hence-and-thus-be-followed-by-commas

Reply:
See 421347ed8ec4d076906dfbafb1c924fd0a9fcf11